src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Network Analysis/FindServiceArea/FindServiceArea.xaml.cs

```diff
@@ -0,0 +1,264 @@
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArcGISRuntime.WPF.Samples.FindServiceArea
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Find Service Area (Interactive)",
+        "Network Analysis",
+        "Demonstrates how to find services areas around a point using the ServiceAreaTask. A service area shows locations that can be reached from a facility based off a certain impedance [such as travel time]. Barriers can also be added which can effect the impedance by not letting traffic through or adding the time is takes to pass that barrier.",
+        "")]
+    public partial class FindServiceArea
+    {     
+        
+        // All location where service areas will be found
+        private List<ServiceAreaFacility> _serviceAreaFacilities;
+        // Task to find service area around a facility
+        private ServiceAreaTask _serviceAreaTask;
+        // Used for solving task above
+        private ServiceAreaParameters _serviceAreaParameters;
+        // For displaying service area facilities to the mapview
+        private GraphicsOverlay _facilityOverlay;
+        // For displaying service areas to the mapview
+        private GraphicsOverlay _serviceAreasOverlay;
+        // For displaying barriers to mapview
+        private GraphicsOverlay _barrierOverlay;
+        // Used to make barriers
+        private PolylineBuilder _barrierBuilder;
+        // Fills service areas with a color when displayed to mapview
+        private List<SimpleFillSymbol> _fillSymbols;
+        // Used for placing geometry on mapview
+        private SpatialReference _spatialReference = SpatialReferences.WebMercator;
+
+        // Symbol for facilities
+        private PictureMarkerSymbol _facilitySymbol;
+
+        // Used for service area outlines
+        private SimpleLineSymbol _outline;
+
+        // Used to see whether the user is placing barriers or facilities
+        private Boolean _barrierMode = false;
+        private Boolean _facilityMode = false;
+
+        // Uri for service area
+        private Uri _sanDiegoServiceAreaUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ServiceArea");
+
+        // Picture for facility symbol
+        private Uri _pictureMarkerUri = new Uri("http://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png");
+
```

Comment:
The variables defined above are spaced nicely. Don't forget periods at the end of comments.